Formula/latex2html.rb

```diff
@@ -20,4 +22,43 @@ def install
                           "--with-texpath=#{share}/texmf/tex/latex/html"
     system "make", "install"
   end
+
+  test do
+    # Trivial Tests
+    assert_match version.to_s, shell_output("#{bin}/latex2html --version")
+    system "#{bin}/latex2html", "--help"
+
+    # Non-Trivial Test
+    test_file = File.new(testpath/"test.tex", "w")
+    contents_of_file = '
+      \documentclass{article}
+      \usepackage[utf8]{inputenc}
+
+      \title{Experimental Setup}
+      \date{November 2016}
+
+      \usepackage{natbib}
+      \usepackage{graphicx}
+
+      \begin{document}
+
+      \maketitle
+
+      \section{Experimental Setup}
+        \textbf{it works!}
+      \end{document}
+    '
+    test_file.puts(contents_of_file)
+    test_file.close
```

Comment:
You’ll have to double every backslack like so:
(testpath/"test.tex").write <<-EOS.undent
  \\documentclass{article}
  \\usepackage{...}
  ...
EOS